keyboards/kamigakushi/info.json

```diff
@@ -0,0 +1,123 @@
+{
+    "manufacturer": "Alabahuy",
+    "keyboard_name": "kamigakushi",
+    "maintainer": "Alabahuy",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "GP17",
+        "on_state": 1
+    },
+    "rgblight": {
+        "pin": "GP25",
+        "led_count": 2,
+        "hue_steps": 8,
+        "saturation_steps": 8,
+        "brightness_steps": 8,
+        "max_brightness": 255,
+    "animations": {
+        "alternating": true,
+        "breathing": true,
+        "christmas": true,
+        "knight": true,
+        "rainbow_mood": true,
+        "rainbow_swirl": true,
+        "rgb_test": true,
+        "snake": true,
+        "static_gradient": true,
+        "twinkle": true
+        }
+    },
+    "matrix_pins": {
+        "cols": ["GP28", "GP5", "GP4", "GP3", "GP2", "GP1", "GP0", "GP14", "GP13", "GP12", "GP11", "GP10", "GP9", "GP8", "GP7" ],
+        "rows": ["GP24", "GP29", "GP6", "GP15", "GP16"]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0765",
+        "vid": "0x414C"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```

Comment:
From PR checklist
> * use of LAYOUT_all is only valid when providing additional layout macros
>     * providing only LAYOUT_all is invalid - especially when implementing the additional layouts within 3rd party tooling

Reply:
can i use LAYOUT_65_ansi_7u for this layout ? @zvecr

Reply:
Referencing <https://github.com/qmk/qmk_firmware/blob/master/layouts/default/readme.md>,


`LAYOUT_65_ansi_blocker_tsangan` matches better?

Reply:
right, thanks :)